docs/integrations/saas-cloud/docusign.md

```diff
@@ -0,0 +1,209 @@
+
+The app leverages the DocuSign Monitor API to collect and analyze data from your DocuSign account, including document access and modification, user activity, authentication activity, and system events. You can set up custom alerts for critical events and user activity, and receive notifications in real-time when potential security threats are detected.
+
+With the DocuSign App, you can:
+
+* Monitor who accesses and changes important documents and detect any instances of unauthorized access or modifications to sensitive documents.
+* Monitor user actions to identify any suspicious behavior, including attempts to access unauthorized data or engage in malicious activity.
+* Monitor login activity to identify any abnormal or suspicious login attempts or unusual patterns of activity.
+* Monitor system events for any signs of anomalies or potential security threats.
+* Prevent data loss by closely monitoring document access, and changes and detecting any unauthorized activity.
+* Analyze user behavior to identify potential security threats by looking for deviations from normal patterns of activity.
+
+The Sumo Logic DocuSign App is designed to help you maintain the security of your DocuSign account and prevent data breaches. With powerful monitoring and analytics capabilities, the app provides actionable insights into user activity and critical alerts, allowing you to identify and address potential security threats before they can cause harm.
+
+## Log Types
+
+The DocuSign App for Sumo Logic uses [Events](https://developers.docusign.com/docs/monitor-api/monitor101/events-alerts/events-list) and [Alerts](https://developers.docusign.com/docs/monitor-api/monitor101/events-alerts/alerts-list) data to generate logs that can be used for monitoring and analysis.
+
+### Sample Events Log Message
+
+```json
+{
+ "timestamp": "2023-03-03T09:10:06.0512813Z",
+ "eventId": "a8010d32-b9f5-4948-9195-50c867d7c3a5",
+ "site": "DEMO",
+ "accountId": "8949bd2a-f1fc-4c95-ae8e-9e294b2af5b7",
+ "organizationId": "9e294b2af5b7-f1fc-4c95-8949bd2a",
+ "userId": "9cfb472b-ef1f-4116-8df2-17c53841fd6c",
+ "integratorKey": "##DOCUSIGN_INTERNAL_NOT_USED##",
+ "userAgent": "",
+ "ipAddress": "162.248.186.11",
+ "ipAddressLocation": {
+   "latitude": 47.6,
+   "longitude": -122.33,
+   "country": "india",
+   "state": "gujarat",
+   "city": "akota"
+ },
+ "object": "Account",
+ "action": "UserAdded",
+ "property": "",
+ "field": "",
+ "result": "",
+ "source": "API",
+ "referencedUserId": "2020ac66-26b5-4367-aff3-f33bd9a007b7",
+ "proxyStatus": "",
+ "proxyType": "",
+ "proxyLevel": "",
+ "latitude": 47.6,
+ "longitude": -122.33,
+ "city": "akota",
+ "state": "gujarat",
+ "country": "india",
+ "data": {
+   "AffectedUserId": "2020ac66-26b5-4367-aff3-f33bd9a007b7",
+   "AffectedMembershipId": "4da49358-8de5-4f62-9e9a-6029259723bc",
+   "PermissionProfileId": "14414475"
+ }
+}
+```
+
+### Sample Events Query
```

Comment:
This should come under the Heading "Sample Queries".  Sample Queries should be at the same level as "Sample Logs". Please check the recent released app docs.

Reply:
Cool. Done.